crates/rpc/rpc/src/eth/bundle.rs

```diff
@@ -269,6 +283,20 @@ where
     }
 }
 
+#[async_trait::async_trait]
+impl<Eth> MevSimApiServer for EthBundle<Eth>
```

Comment:
for EthSimBundle, we want a standalone type, this way we have a nice separation of namespaces